Modules/Files/test/Files.API.Tests.Performance.EndToEnd/src/download-file.test.ts

```diff
@@ -0,0 +1,94 @@
+import http from "k6/http";
+import { Options } from "k6/options";
+import { describe, expect } from "https://jslib.k6.io/k6chaijs/4.3.4.2/index.js";
+
+const host = __ENV.HOST;
+const apiEndpoint = host+"/api/v1";
+
+interface Data {
+    authToken: string,
+    fileId: string
+}
+
+function translateSize() {
+    switch(__ENV.SIZE){
```

Comment:
Did you run prettier? Because I would expect there to be a space between `)` and `{`.

Reply:
So I ran prettier, but I mistyped the command so It didn't do anything. I also installed the extension, but didn't set it up to change on save, so it didn't do anything either. :) It's fixed now.